test/config.py

```diff
@@ -14,6 +14,7 @@
     },
 
     'irods_zone': 'tempZone',
+    'irods_hostname_for_resources': '97f627fe73a6',
```

Comment:
seems... particular.    do we want a hostname here?

Reply:
This was added due to the tests needing to create resources in the docker container running iRODS.

This may not be necessary, but I'm pretty sure it is.

Reply:
okay, just caught my eye as surprising.

Reply:
I'm not surprised. I was not happy when I added that property, but I don't know if there's a better way to do it yet.

Reply:
can we just inject it dynamically?

Reply:
right, because you're removed from the point of activity.    okay... will keep thinking.   carry on.

Reply:
done